### PR TITLE
Refactor jobs/sessions section

### PR DESCRIPTION
**Pull request by jkwmoore**
Hopefully this is going to make the background info and cluster / scheduler specific differences very clear.

Have made a lot of changes.

### CODE REVIEW

**Comment by jkwmoore** [comment]
Pending an update to the readme file.

**Comment by jkwmoore** [comment]
This should now be good to merge with the updated explanation to the readme (important files / folders.)

**Review by desmfryan — COMMENTED** [review]
Hi James

1) Bessemer ls-dyna 'While using a X11 GUI forwarding supported SSH client, an interactive session can be started on ShARC with the srun --pty bash -i command which supports graphical applications' 
- Typo ShARC needs to be replaced with Bessemer.
- srun --pty bash -i needs to be removed since you already deal with launching an interactive session earlier in the section (a link to)

Just looking at qsh-vis onwards (tomorrow).

Des

File: hpc/scheduler/index.rst [hunk 1]
```diff
@@ -10,20 +10,581 @@ Using interactive sessions / submitting batch jobs
 
     ./*
```
Comment (jkwmoore) [inline hpc/scheduler/index.rst:11]: We may wish to manually specify the TOC here to avoid the weird order with DRMAA in the menu.

**Pull request merged**